lib/api.js

```diff
@@ -87,6 +89,22 @@ async function handlePullRequestReviewUpdate(context, eventName, event) {
   }
 }
 
+async function handleCheckUpdate(context, eventName, event) {
+  const { action } = event;
+  const payload = eventName === 'check_suite' ? event.check_suite : event.check_run;
+  if (action === 'completed') {
```

Comment:
I haven't tested this code, but shouldn't this be `action !== 'completed'`?

Reply:
This would be great to have in.  Happy to update the PR.

How can I pull this branch locally or publish a canary build to check it with the update?

Reply:
unfortunately, didn't work for me. i got these messages:

```
INFO  No update necessary
INFO  PR is probably ready: mergeable_state: unknown
INFO  Failed to merge PR: Required status check "continuous-integration/jenkins/branch" is expected.
```

this is after the workflow yml with these _on_ event types:

```
on:
  pull_request:
    types:
      - labeled
      - unlabeled
      - synchronize
      - opened
      - edited
      - ready_for_review
      - reopened
      - unlocked
  pull_request_review:
    types:
      - submitted
  check_suite:
    types:
      - completed
  check_run:
    types:
      - completed
  status: {}
```